src/Umbraco.Web.UI/Umbraco/config/lang/en.xml

```diff
@@ -1288,7 +1288,7 @@ To manage your website, simply open the Umbraco backoffice and start adding cont
     <key alias="installStateComplete">All done, your browser will now refresh, please wait...</key>
     <key alias="installStateCompleted">Please click 'Finish' to complete installation and reload the page.</key>
     <key alias="installStateUploading">Uploading package...</key>
-    <key alias="certifiedToWorkOnUmbracoCloud">Certified to work on Umbraco Cloud</key>
+    <key alias="certifiedToWorkOnUmbracoCloud">Verified to work on Umbraco Cloud</key>

```

Comment:
@AndyButland maybe update the alias as well to `verified`?

Reply:
Agreed, have updated.